docs/Documentation/eagle_to_kestrel_transition.md

```diff
@@ -0,0 +1,144 @@
+
+### How to build your own software on Kestrel
+
+If you need to build your own software on Kestrel, and NOT use an already-existing module, then the steps can be a bit different than Eagle. For a general software-building procedure, please see our [Libraries How-To](/Documentation/Development/Libraries/howto/#summary-of-steps) tutorial.
+
+In general, on Kestrel we recommend using the `PrgEnv-cray` or `PrgEnv-intel` environments to build your code. For detailed descriptions on these environments, see our [environments](/Documentation/Systems/Kestrel/Environments/) page. For a tutorial walkthrough of building a simple code (IMB) within these environments, see our [environments tutorial](/Documentation/Systems/Kestrel/Environments/tutorial/) page. Note that `PrgEnv-` environments on Kestrel are different than environments on Eagle. Loading a `PrgEnv` loads a number of modules at once that together constitute a consistent environment. 
+
+**NOTE**: OpenMPI currently does not work well on Kestrel, and thus it is **strongly** recommended to NOT use OpenMPI. If you require assistance in building your code with an MPI other than OpenMPI, please reach out to hpc-help@nrel.gov. The issue with OpenMPI is at the networking layer, and building your own OpenMPI will not fix the issue.
+
+**NOTE**: Some MPI codes, especially old legacy scientific software, may be difficult to build with Cray MPICH. In these cases, if it is possible to build the code with Intel MPI or a different MPICH implementation, then Cray MPICH can be utilized at run-time via use of the `cray-mpich-abi` module (note that OpenMPI is *NOT* an implementation of MPICH, and you cannot use the `cray-mpich-abi` if you built with OpenMPI). A detailed example of building with Intel MPI but running with Cray MPICH can be found on our [VASP application page](/Documentation/Applications/vasp/). 
+
+## 4. Running your jobs on Kestrel
+
+See our page on submitting jobs on Kestrel [here](/Documentation/Systems/Kestrel/running/).
+
+Submitting a job on Kestrel works much the same as submitting a job on Eagle. Both systems use the Slurm scheduler. If the application you wish to run can be found under our [Applications tab](/Documentation/Applications/), then there may be example Kestrel submit scripts on the application page. Otherwise, our [VASP documentation page](/Documentation/Applications/vasp/#vasp-on-kestrel) contains a variety of sample submit scripts that you can modify to fit your own purposes.
+
+### Shared Partition
+
+Note that each Kestrel CPU node contains 104 CPU cores (and 256 GB memory). Some applications or application use-cases may not scale well to this many CPU cores. In these cases, it is recommended to submit your jobs to the shared partition. A job submitted to the shared partition  will be charged AUs proportionate to whichever resource you require more of, between CPUs and memory.
+
+The following is an example shared-partition submit script using VASP:
```

Comment:
There's an example job submission script and AU calculation in the shared partition section on the running jobs page, so I think it'd be better to just link to that page rather than have the same content twice. I think that's easier to maintain and keep updated. Let me know if you think the information on that page isn't clear or needs more detail.